lib/fpm/package/pacman.rb

```diff
@@ -96,7 +97,7 @@ def dependencies
   def input(pacman_pkg_path)
     control = {}
     # Unpack the control tarball
-    safesystem("tar", "-C", staging_path, "-xf", pacman_pkg_path)
+    safesystem(FPM::Util.tar_cmd, "-C", staging_path, "-xf", pacman_pkg_path)
```

Comment:
No need to namespace here, just `tar_cmd` is fine although that's more of a style thing, same at line 268.